db/records.py

```diff
@@ -299,23 +300,30 @@ def create_records_from_csv(
         escape=None,
         quote=None,
 ):
-    with open(csv_filename, 'rb') as csv_file:
+    with open(csv_filename, "rb") as csv_file:
         with engine.begin() as conn:
             cursor = conn.connection.cursor()
-            relation = '.'.join('"{}"'.format(part) for part in (table.schema, table.name))
-            formatted_columns = '({})'.format(','.join([f'"{column_name}"' for column_name in column_names]))
-
-            copy_sql = f'COPY {relation} {formatted_columns} FROM STDIN CSV'
-            if header:
-                copy_sql += " HEADER"
-            if delimiter:
-                copy_sql += f" DELIMITER E'{delimiter}'"
-            if escape:
-                copy_sql += f" ESCAPE '{escape}'"
-            if quote:
-                if quote == "'":
-                    quote = "''"
-                copy_sql += f" QUOTE '{quote}'"
+            relation = sql.SQL(".").join(
+                sql.Identifier(part) for part in (table.schema, table.name)
+            )
+            formatted_columns = sql.SQL(",").join(
```

Comment:
Do we need to call `sql.SQL` on the `","`? It's only one character.

Reply:
Unfortunately, when we call `sql.Indentifier` which will return a **Identifier** class (based on sql.Composable) instead of a **String**. So we have to convert our `","` to sql.Composable using sql.SQL. 

Otherwise, we'll receive such an error: 
```
 File "/code/db/records.py", line 306, in create_records_from_csv

    relation = ".".join(

TypeError: sequence item 0: expected str instance, Identifier found
```

Reply:
Okay, thanks! It would be useful to add a comment explaining this so that other people reading the code will also know it.

Reply:
Copy that. Thank you kgodey !